docs/background/main.rst

```diff
@@ -1,23 +1,28 @@
-================
-BaPSF Background
-================
+=====================
+BaPSF HDF5 Data Files
```

Comment:
This needs to remain "BaPSF Background". This section contains any background information pertaining to the actually facility and not just the HDF5 files.

We can debate what should go in here and how to make it flow better with the rest of the documentation, but, for now, it should remain as "BaPSF Background".

Reply:
The BaPSF background seems very similar to an About section, where it contains information about what the package is about, the people who developed/maintains the package, the facility for which this was build etc. and I think we can do this in one page. The gritty details could be put on the website if its not directly related to the package.

I'm thinking the documentation could be organized into "what the package does and whats inside it" (i.e. available tools and features) vs "the data files that it is supposed to look at and whats inside _that_" (i.e. how stuff is written to a data file and what information the program is looking for). I was going for a re-categorizing of what's currently available on the documentation but can reinstate the BaPSF background part if necessary.